rust/Cargo.lock

```diff
@@ -2033,9 +2033,9 @@ dependencies = [
 
 [[package]]
 name = "kittycad-modeling-cmds"
-version = "0.2.108"
+version = "0.2.110"
```

Comment:
Can you also update `rust/Cargo.toml`? Usually we update it there since it's the human-readable config. Then run `cargo update` or similar and it updates this file.